### ISSUE TITLE
Question about Implementation Results

### ISSUE THREAD

**Issue by HyoKong** [luminxu/Pose-for-Everything#14]
Hi, thank you for your interesting work!

I collect all the training dataset based on your dataset guidance. The evaluation result, however, is much lower than the one you report in the paper. Could you please check if there is something wrong with your released code? Thank you so much!

![111](https://user-images.githubusercontent.com/32049327/210492687-11b54e4b-640b-471b-ae39-7a79174546df.png)

**Comment by zhanghao5201** [comment]
Have you solved the problem? I'm in the same situation. On the test set, validation set, and training set, the results are low and close to yours.

**Comment by worklikerlei** [comment]
> Have you solved the problem? I'm in the same situation. On the test set, validation set, and training set, the results are low and close to yours.

Have you solved the problem? I'm in the same situation too.

**Comment by zy-meng** [comment]
I met the same situation, the loss is about 0.0003 at start and never decrease, the best PCK is at 10th epoch in the total 210 epochs

**Comment by worklikerlei** [comment]
> I met the same situation, the loss is about 0.0003 at start and never decrease, the best PCK is at 10th epoch in the total 210 epochs
the best PCK is at 20th epoch in the total 210 epochs in split1 train. the test result in split1 only 50+ PCK, how about you?

**Comment by zy-meng** [comment]
> the best PCK is at 20th epoch in the total 210 epochs in split1 train. the test result in split1 only 50+ PCK, how about you?

Did you trained exactly according the paper's configuration? My test result in split1 is only 49+ PCK at 10th epoch.

**Comment by chenbys** [comment]
My results about split1 are around 50 PCK. Is there anyone that can achieve 84.23 stated in paper?

**Comment by yupei-song** [comment]
It would be great if the author could release the training log.

**Comment by jin-s13** [comment]
Thank you for reporting this issue. I appreciate you taking the time to bring this to our attention. Unfortunately, the original author of this repository has since graduated and taken a position in industry. They no longer have plans to maintain the codebase going forward. If anyone in the community would be willing to help resolve this issue, I would be incredibly grateful. I would be happy to merge any fix that is proposed and tested.
